### PR TITLE
Tutorial fixes for 1.9

### PR DESCRIPTION
**Pull request by joel-hamill**
## What are your changes?
Review and tested 1.9 tutorials for accuracy and formatting. 
## What type of changes does your doc introduce?
- [ ] Bug fix
- [x] New feature 

## What is the urgency level of this change?
- [ ] Blocker
- [ ] High
- [x] Medium

## I have completed these items:
- [x] I have tested all commands and code snippets.
- [x] I have built locally and tested for broken links and formatting.
- [ ] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [ ] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

Ping `@emanic`, `@sascala`, or `@joel-hamill` for reviewing pull request changes.

## If you included a comment with your commit, it appears here:

### CODE REVIEW

File: 1.7/usage/tutorials/wordpress-mysql.md [hunk 1]
```diff
@@ -176,7 +176,7 @@ Now that you have your application definitions ready, let's fire up the DC/OS Ma
 
 ## Deploying the Web App via the Marathon Web Interface
 
-From the DC/OS web interface, click the **Services** tab and select **Marathon**.
+From the DC/OS web interface, click the **Services > Services** tab and select **Marathon**.
```
Comment (emanic) [inline 1.7/usage/tutorials/wordpress-mysql.md]: Is this the right path in 1.7?

Comment (joel-hamill) [inline 1.7/usage/tutorials/wordpress-mysql.md]: No, you're right. I'll fix.

File: 1.9/overview/architecture/components.md [hunk 2]
```diff
@@ -614,7 +614,11 @@ DC/OS provides multiple different ways to provision and allocate disk space and
 
 The **Cluster ID service** was removed in DC/OS 1.9.0. The universally unique identifier (UUID) for each cluster is now generated by the DC/OS Setup service.
 
+<<<<<<< HEAD
 The **Mesos Persistent Volume Discovery service** was removed in DC/OS 1.9.0. Detection of [mounted disk resources](https://dcos.io/docs/1.9/administration/storage/mount-disk-resources/) is now handled by the DC/OS Setup service.
+=======
+The **Mesos Persistent Volume Discovery service** was removed in DC/OS 1.9.0. Detection of [mounted disk resources](/docs/1.9/administration/storage/mount-disk-resources/) is now handled by the DC/OS Setup service.
+>>>>>>> master
```
Comment (emanic) [inline 1.9/overview/architecture/components.md]: I think there may be some conflict markers still here or so it seems.

Comment (joel-hamill) [inline 1.9/overview/architecture/components.md]: i searched specifically for `>>>` but go no hits :(. i will fix

**Pull request merged**